### ISSUE TITLE
Purpose of this bundle?

### ISSUE THREAD

**Issue by enumag** [skrz/autowiring-bundle#11]
I understand that this bundle is from the time when Symfony didn't have autowiring, autoregistration and other advanced featured. But symfony does have these features now. From the readme I'm not sure if this bundle still has some value or not in Symfony 4 environment. Can you give me some info about this?

I was looking for an integration of bunny to symfony and your bunny bundle depends on this one so I'd like to know what is it about before I install it.

**Comment by mzstic** [comment]
Sorry for late reply, missed the notification. Unfortunatelly we didn't have time to play with Syfmony 4 yet :/
Things I know about Symfony 4 can't do are
- property Autowiring
- scalar values autowiring

I think more info should be able to provide @jakubkulhan who already used it with symfony 4 and is original author of this bundle :)

**Comment by jakubkulhan** [comment]
Yep, in Symfony 4 this bundle only adds annotation-based property/method autowiring & scalar container parameter injection.

**Comment by enumag** [comment]
Both of these features are a bad practice in my opinion. By bad practice I mean that it should not be specified in the classes by annotations but in config instead. The class is not supposed to know where the values are from or if they are autowired. Therefore I don't want this package in my application.

It might be a good idea to release a new version of BunnyBundle without dependency on this one. I'll probably use EnqueueBundle anyway so it doesn't matter to me but you still might want to consider it.

Thanks for your answers.
